lib/entities/integrations/index.ts

```diff
@@ -19,11 +20,30 @@ export class Integrations {
    */
   public async getAll(planIdentifier: string): Promise<IIntegration[] | undefined> {
       const urlSegments = [planIdentifier, this.baseUrl];
+         return await wrapWithErrorHandler(async () => {
+            const url = buildApiUrl(urlSegments);
+            const res = await this.networkClient.get<IIntegration[]>(url);
+            return res;
+        });
+    }
 
-       return await wrapWithErrorHandler(async () => {
-          const url = buildApiUrl(urlSegments);
-          const res = await this.networkClient.get<IIntegration[]>(url);
-          return res;
-      });
+  /**
+   * Create a new integration for the given plan
+   * @param planIdentifier - Identifier of the target plan
+   * @param integration - The integration to create
```

Comment:
I think the integration param could be more descriptive such as "The integration payload to create a new integration"